### ISSUE TITLE
A11y_Visual Studio Code_Customizations and adding extensions_Keyboard: "Install", "Manage" controls in the Extension page are not accessible using keyboard

### ISSUE THREAD

**Issue by v-kanve** [microsoft/vscode#122847]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code;#Keyboard;#WCAG2.1.1;#Win32

### Environment Details:​
Application Name: Visual Studio Code
Application Version: 1.55.2 (user setup)
Date: 2021-04-13T09:35:57.887Z
Electron: 11.3.0
Chrome:87.04280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Standard Violated: WCAG2.1.1

### Note:
This issue is observed at any extension.

### Repro Steps:​
1. Turn on JAWS
2. Open Visual Studio Code editor
3. Navigate to the extension side view bar option using "Ctrl+Shift+X" key navigate to the extension side view bar option.
4. Tab till search and type "Jupyter" extension
5. Tab till Jupyter extension and press enter
6. Tab till "Install", "Manage" controls using keyboard
7. Verify whether "Install", "Manage" controls are accessible using keyboard or not

### Actual Result:
"Install", "Manage" controls in the Extension page are not accessible using keyboard

### Expected Result: ​
"Install", "Manage" controls in the Extension page should be accessible using keyboard also. After License control when we tab the keyboard focus should go to "Install" and then to "Manage" control and then to the details section.

### User Impact:
Keyboard only users will face difficulty while navigating through the application if any control is not accessible using keyboard. 

### Attachment:
[A11y_Visual Studio Code_Customizations and adding extensions_Keyboard Install, Manage controls in the Extension page are not accessible using keyboard.zip](https://github.com/microsoft/vscode/files/6414549/A11y_Visual.Studio.Code_Customizations.and.adding.extensions_Keyboard.Install.Manage.controls.in.the.Extension.page.are.not.accessible.using.keyboard.zip)

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/120202

**Comment by InduPriya1805** [comment]
GitHubTags:#VisualStudioCode-Win32-Apr2021;
